File: FizzBuzz.js
for(var i = 1; i<=50; i++){
	var n = '';
	if (i % 3 == 0) {
		n = 'Fizz';
	}
	if (i % 5 == 0) {
		n += 'Buzz';
	}
	if (! ( (i % 5 == 0) || (i % 3 == 0) ) ) {
		console.log(i);
	} else {
		console.log(n);
	}
}

File: FizzBuzz.cpp
#include <iostream>

using namespace std;

int main () {
    
    for (int i = 1; i <= 50; i++) {
        
        if (i % 3 == 0) {
        
        	cout << "Fizz";
        
        }
        
        if (i % 5 == 0) {
        
        	cout << "Buzz";
        
        }
        
        if (! ( (i % 5 == 0) || (i % 3 == 0) ) ) {
        
        	cout << i;

        }
        
        cout << endl;
    
    }

}
